lib/bump.rb

```
@@ -59,9 +59,9 @@ def current
 
       def bump(file, current, next_version, options)
         replace(file, current, next_version)
-        if options[:bundle] and under_version_control?("Gemfile.lock")
+        if options[:bundle] and Dir.glob('*.gemspec').any? and under_version_control?("Gemfile.lock")
           bundler_with_clean_env do
-            system("bundle")
+            return ["Bundle error", 1] unless system("bundle")
```

before this silently failed ... not good :D